.github/workflows/release.yml

```diff
@@ -116,23 +116,69 @@ jobs:
     if: github.repository == 'argoproj/argo-events'
     needs: [ build-push-linux-multi, bom ]
     steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set Version
+        run: |
+          if [ ${GITHUB_REF##*/} = master ]; then
+           echo "VERSION=latest" >> $GITHUB_ENV
+          else
+            echo "VERSION=${GITHUB_REF##*/}" >> $GITHUB_ENV
+          fi
+
       - name: Download binaries
         uses: actions/download-artifact@v3
         with:
           name: binaries
           path: dist/
+
       - uses: actions/download-artifact@v3
         with:
           name: sbom.tar.gz
-          path: sbom.tar.gz
+          path: /tmp/sbom.tar.gz
+
+      - name: Registry Login
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAYIO_USERNAME }}
+          password: ${{ secrets.QUAYIO_PASSWORD }}
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@9becc617647dfa20ae7b1151972e9b3a2c338a2b # v2.8.1
+        with:
+          cosign-release: 'v1.13.1'
+
+      - name: Install crane to get digest of image
+        uses: imjasonh/setup-crane@e82f1b9a8007d399333baba4d75915558e9fb6a4
```

Comment:
What is the reason of not to use `imjasonh/setup-crane@v0.1` as mentioned in https://github.com/imjasonh/setup-crane?

Reply:
I used the latest version of the action.  It appears the `docs` were not updated.  Time between versions was only three days, with only some minor changes.   Let me know if you prefer to use `v0.1`

Looks like I should comment the version.

Reply:
Actually there's a `v0.2` available, which is the latest.

https://github.com/imjasonh/setup-crane/tree/v0.2

Reply:
I just double checked.  I have pinned v0.2 with the hash.  Just sent a commit that comments the version being used.